cypress/integration/front_page_spec.js

```diff
@@ -6,6 +6,9 @@ describe('Basic tests', function() {
   // Opens pages, which don't require a certain dataset, resource, organization
   // or other item to exist. Only tests that the url doesn't return an error code
   it('Opens all kinds of various pages', function() {
+    // Ensure platform vocabulary for submit-showcase
+    cy.reset_db();
```

Comment:
This could be moved to before function as it is requirement for the test and not related to the actual test itself.

Reply:
Good call 🤦‍♂️ Done in 45c6deb